sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_manage_custom_models_async.py

```diff
@@ -0,0 +1,89 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_manage_custom_models_async.py
+
+DESCRIPTION:
+    This sample demonstrates how to manage the custom models on your account. To learn
+    how to create and train a custom model, look at sample_train_unlabeled_model.py and
+    sample_train_labeled_model.py.
+USAGE:
+    python sample_manage_custom_models_async.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_FORM_RECOGNIZER_ENDPOINT - the endpoint to your Cognitive Services resource.
+    2) AZURE_FORM_RECOGNIZER_KEY - your Form Recognizer subscription key
+"""
+
+import os
+import asyncio
+
+
+class ManageCustomModelsSampleAsync(object):
+
+    endpoint = os.environ["AZURE_FORM_RECOGNIZER_ENDPOINT"]
+    key = os.environ["AZURE_FORM_RECOGNIZER_KEY"]
```

Comment:
FWIW, in .NET we have a [different name](https://github.com/Azure/azure-sdk-for-net/blob/ffb35459d9b96beb4f9622a9000008c651631766/sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Infrastructure/TestEnvironment.cs#L20) for this environment variable.  Not sure it's worth aligning if you're consistent within Python already.

Reply:
I was keeping these consistent with what we have in text analytics rn. @kristapratico should we switch TA and FR to be consistent with .NET in this regard, I don't have strong feelings either way.

Reply:
I think it's fine if we're consistent with python text analytics